electron/process_manager.ts

```diff
@@ -124,6 +126,7 @@ function startLocalShadowsocksProxy(
     ssLocalArgs.push('-p', '' + serverConfig.port);
     ssLocalArgs.push('-k', serverConfig.password || '');
     ssLocalArgs.push('-m', serverConfig.method || '');
+    ssLocalArgs.push('-t', SS_LOCAL_TIMEOUT_SECS.toString());
```

Comment:
Is `toString()` locale dependent? Will we get a localized string in farsi?

Reply:
I did some testing and it seems like `toString` is locale independent (`toStringLocale` allows specifying the locale). We also use it to convert the local port number to string and it seems to work fine.

I can use string interpolation if you think that's better.